packages/components/toast/toast.component.scss

```
@@ -28,8 +28,17 @@ $tokens: meta.module-variables(tokens) !default;
     border-radius: kbq-css-variable(toast-size-border-radius);
 
     margin-right: kbq-css-variable(toast-size-icon-margin-right);
-
-    padding-top: kbq-css-variable(toast-size-content-padding-top);
+    padding-top: calc(
+        var(--toast-size-content-padding-top, map.get($tokens, toast-size-content-padding-top)) +
+            (
+                (
+                        var(--toast-font-title-line-height, map.get($tokens, toast-font-title-line-height)) - var(
+                                --toast-size-icon-width,
+                                map.get($tokens, toast-size-icon-width)
+                            )
+                    ) / 2
+            )
+    );
```

Можно сделать так, чтобы сохранить обертку над css переменными

```suggestion
    $padding-top: kbq-css-variable(toast-size-content-padding-top);
    $toast-font-title-line-height: kbq-css-variable(toast-font-title-line-height);
    $toast-size-icon-width: kbq-css-variable(toast-size-icon-width);
    padding-top: calc(
       $padding-top +
            (
                ($toast-font-title-line-height - $toast-size-icon-width) / 2
            )
    );
```

---

да, супер предложение, поправил